common/include/opae/types_enum.h

```diff
@@ -144,14 +144,12 @@ enum fpga_reconf_flags {
 	FPGA_RECONF_FORCE = (1u << 0)
 };
 
-/**
- * Object Synchronization flags
- *
- * These flags can be passed to the fpgaObjectRead() function.
- */
 enum fpga_object_read_flags {
-	/** Synchronize the object contents by reading from the underlying object. */
-	FPGA_OBJECT_SYNC = (1u << 0)
+  FPGA_OBJECT_SYNC = (1u << 0), /**< Synchronize data from driver */
+  FPGA_OBJECT_GLOB = (1u << 1), /**< Treat names as glob expressions */
+  FPGA_OBJECT_TEXT = (1u << 2), /**< Parse or convert numeric data as text */
```

Comment:
when this flag is set, should we use the (*fpgaObjectRead)(...) to parse the numeric data as text?

Reply:
This flag is only checked in the fpgaObjectRead64 and fpgaObjectWrite64 functions. fpgaObjectRead is used to read raw bytes without parsing it as ascii text.